code/modules/status_system/statusFoodBuffs.dm

```diff
@@ -302,7 +302,8 @@
 	proc/cast()
 
 		var/turf/T = get_step(owner,owner.dir)
-		T = get_step(T,owner.dir)
+		while(get_dist(owner,T) < range)
```

Comment:
Can we cap this? I don't want to lock the whole game if this loop gets fucked

Reply:
this work?